configure.ac

```diff
@@ -2133,6 +2133,17 @@ if test "${CHIP_DEVICE_LAYER_TARGET_LINUX}" = 1; then
         AC_DEFINE([CHIP_WITH_GIO], [0], [Define to 0 to build CHIP without GIO])
     ]
     )
+
+    PKG_CHECK_MODULES(GIOUNIX, gio-unix-2.0 >= 2.28, dummy=yes,
+                AC_MSG_ERROR(GIO Unix >= 2.28 is required))
+
+    AC_SUBST(GIOUNIX_CFLAGS)
+    AC_SUBST(GIOUNIX_LIBS)
+
+    AX_CHECK_COMPILER_OPTIONS([C],   ${GIOUNIX_CFLAGS})
+    AX_CHECK_COMPILER_OPTIONS([C++], ${GIOUNIX_CFLAGS})
```

Comment:
I think you should be able to delete those four lines, PKG_CHECK_MODULES should take care of that. Just like GIO

    AC_SUBST(GIOUNIX_CFLAGS)
    AC_SUBST(GIOUNIX_LIBS)

    AX_CHECK_COMPILER_OPTIONS([C],   ${GIOUNIX_CFLAGS})
    AX_CHECK_COMPILER_OPTIONS([C++], ${GIOUNIX_CFLAGS})

Reply:
Yep, done, thanks